kpi/tests/api/test_api_environment.py

```diff
@@ -103,6 +103,24 @@ def test_template_context_processor(self):
         result = template.render(context)
         self.assertEqual(result, constance.config.TERMS_OF_SERVICE_URL)
 
+    def test_metadata_label_with_default_language(self):
+        """
+        Test that the label field of the user and project metadata contains a default value
+        """
+        def check_metadata_for_label(metadata):
+            for item in metadata:
+                if 'label' in item.keys():
+                    assert 'default' in item['label'].keys()
+                
+        self.client.logout()
+        response = self.client.get(self.url, format='json')
+        assert response.status_code == status.HTTP_200_OK
+        user_metadata = response.data['user_metadata_fields']
+        project_metadata = response.data['project_metadata_fields']
+        check_metadata_for_label(user_metadata)
+        check_metadata_for_label(project_metadata)        
```

Comment:
I'm not sure I understand the purpose of this test. It looks like it only operates on our hard-coded default values. Could you explain under what circumstances this test would fail?

Reply:
No test is probably better than an ineffective test that lends a false sense of security, but we probably should have unit tests for this feature

Reply:
We're just passing through JSON from Constance to the `/environment` endpoint in this PR, and that doesn't really need a test. I may want to manipulate the JSON so that it only returns the label for the currently-active language, but I can do that in another PR. That work would need unit tests.